src/Makefile.am

```diff
@@ -390,13 +395,15 @@ zcashd_LDADD += \
   $(SSL_LIBS) \
   $(CRYPTO_LIBS) \
   $(MINIUPNPC_LIBS) \
+  $(EVENT_PTHREADS_LIBS) \
+  $(EVENT_LIBS) \
   $(LIBZCASH) \
   $(LIBBITCOIN_CRYPTO) \
   $(LIBZCASH_LIBS)
 
 # bitcoin-cli binary #
 zcash_cli_SOURCES = bitcoin-cli.cpp
-zcash_cli_CPPFLAGS = $(BITCOIN_INCLUDES)
+zcash_cli_CPPFLAGS = $(BITCOIN_INCLUDES) $(EVENT_CFLAGS)
```

Comment:
Checked with upstream, and they don't have `$(EVENT_PTHREADS_CFLAGS)` in current master, so unlikely I made a mistake while cherry-picking. I wouldn't expect it to be necessary anyway, as the threading side should only be needed in `zcashd`.